finalPrep/README.md

```diff
@@ -23,51 +23,54 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
 
 ## Gearing Up for Gear Ups
 
 Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry. 
 
-- [ ] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
-  - [ ] Add a link for your gist here: 
+- [x] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
+  - [x] Add a link for your gist here: https://gist.github.com/Jlavarine/3feaf6d66e11b669eb28c1192a4f0500
 
 ## Time Management Prep
 
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
 
-- [ ] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Front End curriculum site](https://frontend.turing.edu/))
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [x] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Front End curriculum site](https://frontend.turing.edu/))
+- [x] Study/Project work time (usually about 25 hours/week outside of class time)
+- [x] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here` <img width="938" alt="Screen Shot 2022-01-10 at 2 53 46 PM" src="https://user-images.githubusercontent.com/96446170/148838124-a2e8db2d-aa15-4905-af55-077f9f248b22.png">
 
+- `Add Week 2 Screenshot Here` <img width="1098" alt="Screen Shot 2022-01-10 at 2 53 57 PM" src="https://user-images.githubusercontent.com/96446170/148838167-eb55d976-c2b1-424f-9c17-f35fe3daafd1.png">
+
+- `Add Week 3 Screenshot Here` <img width="1098" alt="Screen Shot 2022-01-10 at 2 54 05 PM" src="https://user-images.githubusercontent.com/96446170/148838195-650fd54e-51b7-4842-a4d5-e4f807708c69.png">
+
+`Calendar Note` In the description section for the HW & Practice, I have specifics on what I plan to work on during that time. 
```

Comment:
Good note, and good template to start from during Mod 1! Don't hesitate to make changes as your schedule inevitably changes with group work, personal stuff, etc.